app/jobs/reports/authentication_report.rb

```diff
@@ -0,0 +1,43 @@
+require 'reporting/authentication_report'
+
+module Reports
+  class AuthenticationReport < BaseReport
+    REPORT_NAME = 'authentication-report'
+
+    attr_accessor :report_date
+
+    def perform(report_date)
+      return unless IdentityConfig.store.s3_reports_enabled
+
+      self.report_date = report_date
+      message = "Report: #{REPORT_NAME} #{report_date}"
+      subject = "Weekly Authentication Report - #{report_date}"
+
+      report_configs.each do |report_hash|
+        tables = weekly_authentication_report_csv(report_hash['issuers'])
```

Comment:
I think this has to be wrapped as an array
```suggestion
        tables = [weekly_authentication_report_csv(report_hash['issuers'])]
```

Reply:
thank you! i tested it locally (thank you also for showing me how to do that) and it does

Reply:
added in https://github.com/18F/identity-idp/pull/9295/commits/9164075c4b81d0f715372859b7eb9e0f92d44513